components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/saveInbound.jsp

```diff
@@ -47,7 +47,7 @@
 			List<ParamDTO>sParams = new ArrayList<ParamDTO>();
 			Map<String,String[]>paramMap = request.getParameterMap();
 			for(String strKey:paramMap.keySet()){
-				if(strKey.startsWith("transport.") || strKey.startsWith("java.naming.") || strKey.startsWith("inbound.") || strKey.startsWith("api.") || strKey.startsWith("dispatch.filter.")){
```

Comment:
@prabathariyaratna : Please remove the code changes which are introduced due to formatting.